release.md

```diff
@@ -0,0 +1,22 @@
+## One place for release configuration
+
+Release captains should edit a single file to update any configuration values.
+Those values typically need to have set versions before each release.
+They are found in different places, like Dockerfiles, Go files, etc.
+
+The `config.yaml` is the single source of truth. It lists all values by name and value.
+Changes to it should be committed to source control.
+
+### Update an existing value
+
+Edit the config.yaml file and update the desired value.
+Then run any relevant scripts.
+
+### Add a new value
+
+Add it to config.yaml.
+Then update the `config` script. For example:
```

Comment:
Would also be good to include a general guide on updating the places that it is used - the below will create a new build var but it won't actually get used when building rancher.